wpbatch.php

```diff
@@ -200,15 +200,51 @@ private function wpRestore()
         $this->jsonRead();
         $this->databaseCredentialsDefine();
         $this->adminDefine();
-        $this->databaseCreate();
+        $this->databaseRestore();
         $this->wpDownload();
         $this->wpConfigCreate();
         $this->wpInstall();
+        $this->wpConnect();
         $this->domainChange();
         $this->themesInstall();
         $this->pluginsInstall();
     }
 
+    /**
+     * Restores database from source sql file
+     */
+    private function databaseRestore()
+    {
+        $serverName = self::$config['database']['host'];
+        $userName = self::$config['database']['user'];
+        $password = self::$config['database']['password'] ?? '';
+        $dbName = self::$config['database']['name'];
+        $dbSource = self::$config['database']['source'];
+
+        $conn = new mysqli($serverName, $userName, $password);
+        if ($conn->connect_error) {
+            die("Connection failed: " . $conn->connect_error);
+        }
+
+        $sql = 'DROP DATABASE '.$dbName;
+        $conn->query($sql);
+
+        $sql = 'CREATE DATABASE '.$dbName;
```

Comment:
1. I'd suggest using $db variable instead of $conn
2. Restore mode should be selected via function parameter (and it seems we need class constants for different modes)

The modes are:
self::DROP_IF_EXISTS - drop a database, create a new one
self::CREATE_IF_NOT_EXISTS (should be default) - create a new db if it does not exist, or use existing db with no modifications
self::CLEAR_IF_EXISTS - clear DB contents if db exists (should be complimentary with  CREATE_IF_NOT_EXISTS)
self::DO_NOTHING - if db exists, just start restore procedure, die otherwise

3. Spaces should be added around concatenation operation